tasks/main.yml

```diff
@@ -10,21 +10,25 @@
   apt: name=munin-node state=present
   when: ansible_os_family == 'Debian'
 
+- name: Ensure munin-node is installed (OpenBSD).
+  community.general.openbsd_pkg: name=munin-node state=present
```

Comment:
This may break compatibility with Ansible 2.9 (not 100% sure) if not using collections, and will definitely break 2.8 (still *technically* supported by this role). Does `openbsd_pkg` work?

Reply:
You're right, I tested again in a clean environment with Ansible 2.9 and it fails. I think I installed the community collection some time ago and forgot about it.

It works with plain `openbsd_pkg`, so I will switch to that. Ansible 2.10 seems to properly redirect the call to the collection:

```
TASK [munin-node : Ensure munin-node is installed (OpenBSD).] *****************************************************************************************************************************************************
task path: /root/ansible/roles/munin-node/tasks/main.yml:13
redirecting (type: modules) ansible.builtin.openbsd_pkg to community.general.openbsd_pkg
```

As a side-note, with the collection installed in `~/.ansible/collections/`, it works with Ansible 2.9 (the version I'm using in production), but also with Ansible 2.8 (to my surprise):

```
TASK [munin-node : Ensure munin-node is installed (OpenBSD).] *****************************************************************************************************************************************************
task path: /root/ansible/roles/munin-node/tasks/main.yml:13
Using module file /root/.ansible/collections/ansible_collections/community/general/plugins/modules/openbsd_pkg.py
```

It does fail with a syntax error with Ansible 2.7.

Reply:
Fixed